DrinkManagerWeb/Views/Drinks/Create.cshtml

```diff
@@ -0,0 +1,141 @@
+                    <select class="form-control" asp-for="AlcoholicInfo">
+                        <option>Alcoholic</option>
+                        <option>Non alcoholic</option>
+                        <option>Optional alcohol</option>
+                    </select>
+                </div>
+                <div class="form-group col-md-6">
+                    <label asp-for="GlassType">Glass type</label>
+                    <input type="text" class="form-control" asp-for="GlassType" placeholder="Glass type">
+                    <span asp-validation-for="GlassType" class="text-danger"></span>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6">
+                    <div class="form-group">
+                        <label asp-for="Instructions">Instructions</label>
+                        <textarea class="form-control mb-3" rows="3" asp-for="Instructions"></textarea>
+                        <span asp-validation-for="Instructions" class="text-danger"></span>
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="ImageUrl">Image Url</label>
+                        <input type="text" class="form-control" asp-for="ImageUrl" placeholder="Image Url">
+                        <span asp-validation-for="ImageUrl" class="text-danger"></span>
+                    </div>
+                </div>
+                <div class="form-group col-md-6">
+                    <div id="ingredients-group">
+                        <label>Ingredients</label>
+                        <div class="form-group mb-3">
+                            <input type="text" class="form-control" asp-for="Ingredient1" placeholder="Ingredient">
+                            <span asp-validation-for="Ingredient1" class="text-danger"></span>
+                        </div>
+                    </div>
+                    <button type="button" onclick="AddInputAndRestartValidation()" class="btn btn-primary"><i class="fas fa-plus"></i> Add Ingredient</button>
+                </div>
+            </div>
+            <div class="mt-3 border-top">
+                <input class="btn btn-primary mt-3" type="submit" value="Save" />
+            </div>
+        </form>
+    </div>
+</div>
+
+
+
+
+@section Scripts {
+    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
+
+    <script>
+        function AddIngredientInput(value = "") {
+            let ingredientNumber = $("#ingredients-group input").length + 1;
+            if (ingredientNumber > 15) {
+                if ($(".alert").length === 0) {
+                    $("form").prepend(
+                        `<div class="alert alert-danger alert-dismissible" role="alert">
+                                        <button type="button" class="close" data-dismiss="alert">&times;</button>
+                                        You can add maximum of 15 ingredients
```

Comment:
Missing dot in the end of this sentence. Is it possible for this information to appear under the button "Add Ingredient"?

Reply:
done